src/macaron/output_reporter/templates/macaron.html

```diff
@@ -164,6 +164,30 @@
 {% endcall %}
 {% endmacro %}
 
+{% macro render_run_checks(check_tree, excluded_checks) %}
+<ul class="tree-view-nested-list">
+    {% for key, first_level in check_tree.items() %}
+    <li>
+        <div>{{ key }}</div>
+        <ul>
+            {% for key, next_level in first_level.items() recursive %}
+            <li>
+                {% if key in excluded_checks %}
+                <div class="check-tree-node-excluded">{{ key }}</div>
```

Comment:
`class="check-tree-node-excluded"` would give the content a ~strikethrough~ font style while `class="check-tree-node"` don't